src/qt/walletmodel.cpp

```diff
@@ -215,11 +215,6 @@ WalletModel::SendCoinsReturn WalletModel::prepareTransaction(WalletModelTransact
         return OK;
     }
 
-    if(isAnonymizeOnlyUnlocked())
-    {
-        return AnonymizeOnlyUnlocked;
-    }
-
```

Comment:
This should never be the case given the logic, but I guess it worth it to add some kind of final check here 👍  Addressed in 57c4c1b.